pretalx_downstream/templates/pretalx_downstream/settings.html

```diff
@@ -8,6 +8,12 @@ <h2>{% trans "Set up upstream sync" %}</h2>
     <form method="post">
         {% csrf_token %}
         {% bootstrap_form form layout='event' %}
+{% if request.event.settings.upstream_last_sync %}
+        <p>
+            <!-- FIXME format this correctly -->
```

Comment:
Two ways to format this correctly: Either teach hierarkey that this is a datetime object, or pass a formatted string or a datetime object to the view.